charms/knative-eventing/config.yaml

```diff
@@ -3,7 +3,7 @@
 
 options:
   version:
-    default: "1.12.3"
+    default: "1.12.6"
```

Comment:
Should this be 1.12.4?

Reply:
This is according to the [Kubeflow component versions](https://github.com/kubeflow/manifests/tree/1.9.0-rc.2?tab=readme-ov-file#kubeflow-components-versions) section for 1.9.0-rc.2.

Reply:
Ah, I did not notice! Thanks for the info (: